src/core/editor.coffee

```diff
@@ -109,6 +109,19 @@ class Editor
       top: bounds.top - containerBounds.top
     }
 
+  scrollCursorIntoView: () ->
```

Comment:
Can you move this into selection.coffee and just call scrollIntoView? I'm trying to start grouping API calls into selection, editor, document, etc and only having quill.coffee have passthroughs into the most common ones.